### PR TITLE
EZ-599: Rename Cards View Mode as 1xn and 3xn cards

### PR DESCRIPTION
**Pull request by saurabh-2k17**

### CODE REVIEW

File: modules/ezcontent_block/modules/ezcontent_block_cards/ezcontent_block_cards.install [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+/**
+ * @file
+ * Contains update hook.
+ */
+
+/**
+ * Update cards view mode field widget.
+ */
+function ezcontent_block_cards_update_8001() {
```
Comment (arshadkhan53) [inline modules/ezcontent_block/modules/ezcontent_block_cards/ezcontent_block_cards.install:11]: Hey @saurabh-2k17 updates are failing , please see.

**Review by arshadkhan53 — CHANGES_REQUESTED** [review]

**Review by Vinay15 — CHANGES_REQUESTED** [review]
Please update the view_mode_selection field widget for gallery paragraph too.

**Review by Vinay15 — APPROVED** [review]
LGTM

**Review by arshadkhan53 — APPROVED** [review]
LGTM

**Pull request merged**